index.html

```diff
@@ -20,8 +20,8 @@
                         <div class="main-info">
                             <h1 id="main-h1">Monero</h1>
                             <h2 id="main-h2">Private Digital Currency</h2>
-                            <p id="main-text">Monero is a secure, private, untraceable currency. It is open-source and freely available to all. With Monero, you are your own bank. Only 
-                            you control and are responsible for your funds; your accounts 
+                            <p id="main-text">Monero is a secure, private and untraceable cryptocurrency. It is open-source and accessible to all. With Monero, you are your own bank. Only 
```

Comment:
Actually this needs an Oxford comma. "Monero is a secure, private, and untraceable cryptocurrency."